packages/medusa/src/api/routes/admin/store/get-store.ts

```diff
@@ -60,6 +62,7 @@ export default async (req, res) => {
   const storeService: StoreService = req.scope.resolve("storeService")
 
   const featureFlagRouter: FlagRouter = req.scope.resolve("featureFlagRouter")
+  const modulesHelper: ModulesHelper = req.scope.resolve("modulesHelper")
```

Comment:
comment(non blocking): At one point it would be nice to get an enum with everything resolvable to help the user to know what can be resolved and not have to look into the code or the entire doc? it is a global comment and not link to that pr specifically

Reply:
This is an excellent suggestion!